wellregistry/registry/models.py

```diff
@@ -199,7 +199,7 @@ class MonitoringLocation(models.Model):
     alt_method = models.CharField(max_length=300, blank=True, verbose_name='Altitude method')
     alt_acy = models.CharField(max_length=300, blank=True, verbose_name='Altitude accuracy')
 
-    well_depth = ArbitraryDecimalFields(null=True, blank=False)
+    well_depth = ArbitraryDecimalFields(null=True, blank=True)
```

Comment:
Whenever you change the model you should rerun the migrations and update the repo.

Reply:
Thanks for the advice.. I was not aware